contributed/matching_fof_mlz_blending.ipynb

```diff
@@ -0,0 +1,805 @@
+    "index=np.random.choice(len(truth_z12),len(truth_z12),replace=False)\n",
+    "\n",
+    "for i in index[0:len(index)//2]:\n",
+    "    if not np.any(np.isnan([truth_z12[i],coadd_mag_i12[i], coadd_mag_u12[i], \n",
+    "                            coadd_mag_r12[i], coadd_mag_g12[i], coadd_mag_y12[i],\n",
+    "                            coadd_mag_z12[i], mag_error_i12[i], mag_error_u12[i], \n",
+    "                            mag_error_r12[i], mag_error_g12[i], mag_error_y12[i], \n",
+    "                            mag_error_z12[i]])):\n",
+    "        string='%.6f %.5f %.5f %.5f %.5f %.5f %.5f %.5f %.5f %.5f %.5f %.9f %.9f %.9f %.9f %.9f %.9f\\n'%(truth_z12[i],coadd_mag_u12[i],coadd_mag_g12[i],coadd_mag_r12[i],\n",
+    "                                                                                               coadd_mag_i12[i],coadd_mag_y12[i],coadd_mag_z12[i],coadd_mag_u12[i]-coadd_mag_g12[i], \n",
+    "                                                                                               coadd_mag_g12[i]-coadd_mag_r12[i],coadd_mag_r12[i]-coadd_mag_i12[i], \n",
+    "                                                                                               coadd_mag_i12[i]-coadd_mag_z12[i],mag_error_u12[i], mag_error_g12[i],\n",
+    "                                                                                               mag_error_r12[i], mag_error_i12[i],mag_error_y12[i],mag_error_z12[i])\n",
+    "        trainingfile.write(string)\n",
+    "trainingfile.close()\n",
+    "\n",
+    "for i in index[len(index)//2+1:len(index)]:\n",
+    "    if not np.any(np.isnan([truth_z12[i],coadd_mag_i12[i], coadd_mag_u12[i], \n",
+    "                            coadd_mag_r12[i], coadd_mag_g12[i], coadd_mag_y12[i],\n",
+    "                            coadd_mag_z12[i], mag_error_i12[i], mag_error_u12[i], \n",
+    "                            mag_error_r12[i], mag_error_g12[i], mag_error_y12[i], \n",
+    "                            mag_error_z12[i]])):\n",
+    "        string='%.6f %.5f %.5f %.5f %.5f %.5f %.5f %.5f %.5f %.5f %.5f %.9f %.9f %.9f %.9f %.9f %.9f\\n'%(truth_z12[i],coadd_mag_u12[i],coadd_mag_g12[i],coadd_mag_r12[i],\n",
+    "                                                                                               coadd_mag_i12[i],coadd_mag_y12[i],coadd_mag_z12[i],coadd_mag_u12[i]-coadd_mag_g12[i], \n",
+    "                                                                                               coadd_mag_g12[i]-coadd_mag_r12[i],coadd_mag_r12[i]-coadd_mag_i12[i], \n",
+    "                                                                                               coadd_mag_i12[i]-coadd_mag_z12[i],mag_error_u12[i], mag_error_g12[i],\n",
+    "                                                                                               mag_error_r12[i], mag_error_i12[i],mag_error_y12[i],mag_error_z12[i])\n",
+    "        testfile.write(string)\n",
+    "testfile.close()"
```

Comment:
So I see that the code can generate the files, but would there be a quick way to visualize the redshift with respect to some input column (i-band mag?) for these training sets?